src/main/java/spireMapOverhaul/zones/beastslair/BeastsLairZone.java

```diff
@@ -0,0 +1,142 @@
+import com.badlogic.gdx.graphics.Color;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch2;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import com.megacrit.cardcrawl.helpers.MonsterHelper;
+import com.megacrit.cardcrawl.map.MapRoomNode;
+import com.megacrit.cardcrawl.monsters.MonsterGroup;
+import com.megacrit.cardcrawl.random.Random;
+import com.megacrit.cardcrawl.rewards.RewardItem;
+import com.megacrit.cardcrawl.rooms.MonsterRoomElite;
+import spireMapOverhaul.abstracts.AbstractZone;
+import spireMapOverhaul.zoneInterfaces.EncounterModifyingZone;
+import spireMapOverhaul.zoneInterfaces.RewardModifyingZone;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+
+public class BeastsLairZone extends AbstractZone implements EncounterModifyingZone, RewardModifyingZone {
+    public static final String ID = "BeastsLair";
+    public static final String SaveID = makeID("BeastsLairBosses");
+    public static ArrayList<String> bossList = new ArrayList<>();
+
+    public BeastsLairZone() {
+        super(ID, Icons.MONSTER);
+        this.width = 1;
+        this.maxWidth = 1;
+        this.height = 1;
+        this.maxHeight = 1;
+        iconTexture = ImageMaster.MAP_NODE_ELITE;
+
+    }
+
+    public static void clearBossList() {
+        bossList.clear();
+    }
+
+
+    public static void initializeSaveFields() {
+        BaseMod.addSaveField(SaveID, new CustomSavable<ArrayList<String>>() {
+            @Override
+            public ArrayList<String> onSave() {
+                return BeastsLairZone.bossList;
+            }
+
+            @Override
+            public void onLoad(ArrayList<String> strings) {
+                if (strings != null) {
+                    BeastsLairZone.bossList = strings;
+                }
+            }
+        });
+    }
+
+    @Override
+    protected boolean canIncludeTreasureRow() {
+        return false;
+    }
+
+    @Override
+    protected boolean canIncludeFinalCampfireRow() {
+        return false;
+    }
+
+    @Override
+    protected boolean canIncludeEarlyRows() {
+        return false;
+    }
+
+    @Override
+    public void manualRoomPlacement(Random rng) {
+        for (MapRoomNode node : nodes) {
+            node.setRoom(new MonsterRoomElite());
+        }
+    }
+
+    @Override
+    public MonsterGroup changeEncounter(MonsterGroup monsterGroup) {
+        if (bossList.isEmpty()) {
+            return monsterGroup;
+        }
+
+        return MonsterHelper.getEncounter(bossList.get(AbstractDungeon.monsterRng.random(bossList.size() - 1)));
+    }
+
+    @Override
+    public int changeRareCardRewardChance(int rareCardRewardChance) {
+        return 50;
+    }
+
+    @Override
+    public int changeUncommonCardRewardChance(int uncommonCardRewardChance) {
+        return 50;
+    }
+
+    @Override
+    public ArrayList<RewardItem> getAdditionalRewards() {
```

Comment:
I like the theme for this zone, it's something that hasn't been done yet in this mod, and having a 1x1 zone is cool, so I'm on board with the overall concept. I do have some balance feedback, since I've actually made a fight exactly like this before (as part of an event for my act mods). What I discovered from extensive playtesting is that in most runs it's not very difficult to defeat a boss from the previous act.

Compared to when you'd normally face that boss, you have an extra boss relic plus whatever cards, relics, card removals, etc. you've gotten in the meantime. That makes a fight like this easier than an elite, given how bosses are designed (act 1 bosses are largely about achieving consistent damage and block, act 2 bosses are largely about scaling, aoe, and handling enemies with gimmicks). Think about act 2, for example. Which do you think is more likely to kill you -- Slavers or Guardian? In a typical run in act 2, the former is one of the most dangerous fights in the game, the latter is barely a threat.

That means right now, the risk/reward for this is off, since it's a fight that will typically be easier in return for greater rewards. For comparison, the event I mentioned gave the boss 2 additional strength and 25% more HP and the reward was a rare relic plus the normal card/gold rewards (but the rare relic is the only relic and no rarity adjustment to the cards). I felt that was reasonably balanced but still favorable for the player, meaning you wanted to take the fight every time unless your deck was really weak.

So this needs some tuning. The question is whether to do that by increasing the challenge, decreasing the rewards, or some of both. I think the flavor of this zone being a challenging encounter makes sense, so I support increasing the difficulty of this fight until it genuinely is a challenge. To justify the current rewards, though, you'd need to make this fight as challenging as the Colosseum event, since the current rewards are on a similar level (depending on how you value the uncommon/rare cards in the rewards vs. the extra gold from Colosseum).

That suggests one of two approaches:
* Increase the challenge of this fight moderately, so that it's as challenging as a normal elite or a bit more, and reduce the rewards to fit that. You could go with something similar to the 2 strength and 25% HP that I mentioned and tone down the rewards be a bit more than the normal amount for an elite.
* Increase the challenge of this fight immensely, so that it's as challenging as the Colosseum, and keep the current rewards. That means substantially more buffs than the 2 strength and 25% HP that I mentioned. I'm not sure how far you'd need to go, but I'd assume this would require at least 4 strength and maybe some other buffs too (like a bit of artifact or some metallicize). 

This can be adjusted in playtesting, so we don't need to get it exactly correct right now, but it's good to be in the right ballpark. That said, it's a real downside for the second option that it's harder to tune the challenge correctly.

Hope that all makes sense and happy to discuss further.

Reply:
I want to make it feel like a real challenge, so I think I'll try and make it more colosseum level. I'll experiment with different buffs.

Reply:
The problem with a strength buff is that some bosses benefit significantly more from strength than others (imagine Hexaghost with an extra 4 strength on its divider or inferno, or guardian with 4 strength on its whirlwind). Meanwhile slime boss only has a single-hit attack, and loses the strength and other buffs as soon as it splits.

I think instead of strength it would be better to have a % damage increase buff (like this creature deals 33% more damage). Also any new enemies that spawn over the course of the fight should also obtain the buffs.

Reply:
That's true, I'll try that. honestly I might even try things that aren't even straight damage increases